addons/website/views/snippets/snippets.xml

```diff
@@ -1253,6 +1253,14 @@
                        data-customize-website-variable="'border-bottom'">Border Bottom</we-button>
         </we-select>
 
+        <we-select string="Additional colors" data-reload="/">
+            <we-button data-customize-website-views="">Default</we-button>
+            <we-button data-customize-website-views="website.template_header_default_additional_color_primary, website.template_header_mobile_additional_color_primary, website.template_header_hamburger_additional_color_primary, website.template_header_sales_two_additional_color_primary, website.template_header_sales_four_additional_color_primary, website.template_header_sidebar_additional_color_primary, website.template_header_boxed_additional_color_primary
+                ">Primary</we-button>
+            <we-button data-customize-website-views="website.template_header_default_additional_color_secondary, website.template_header_mobile_additional_color_secondary, website.template_header_hamburger_additional_color_secondary, website.template_header_sales_two_additional_color_secondary, website.template_header_sales_four_additional_color_secondary, website.template_header_sidebar_additional_color_secondary, website.template_header_boxed_additional_color_secondary
+                ">Secondary</we-button>
+        </we-select>
```

Comment:
Seems weird: don't we have more header templates than this? :thinking: I think we have more than 10, does not this mean that this option does nothing for half of them?

Reply:
Indeed, some templates are not impacted by this option. But the mobile template is always triggered regardless of the desktop template chosen, so this one will always impacted by this option :).

Reply:
Oh, yes, indeed... same old problem but ok I guess :grimacing: